optin/reporting/test/utils/Validation.spec.js

```diff
@@ -68,42 +70,25 @@ const runBasicPaginationValidationTests = validator => {
 		// Assert:
 		expect(errors.length).to.equal(0);
 	});
-
-	it('should return error missing required params', async () => {
-		// Arrange:
-		const req = TestUtils.mockRequest({});
-		const res = TestUtils.mockResponse();
-
-		// Act:
-		await mockExpressValidatorMiddleware(req, res, validator);
-		const { errors } = validationResult(req);
-
-		const paramPageSize = errors.filter(err => 'pageSize' === err.param)[0];
-		const paramPageNumber = errors.filter(err => 'pageNumber' === err.param)[0];
-
-		// Assert:
-		expect(paramPageSize.msg).to.equal('pageSize required');
-		expect(paramPageNumber.msg).to.equal('pageNumber required');
-	});
 };
 
 const runBasicSortDirectionValidationTests = (invalidParams, validParams, validator) => {
 	// Arrange:
 	const queryField = 'sortDirection';
 
-	it('should return error given incorrect sortDirection', async () => {
-		await runInvalidAsserts({
+	it('fails when sortDirection parameter is omitted', async () => {
+		await runAsserts({
 			validator,
-			invalidParams,
+			values: invalidParams,
 			queryField,
 			errorMessage: 'sort direction is not supported'
 		});
 	});
```

Comment:
i think in some of these tests, the parameters aren't omitted, but rather invalid?

Reply:
ah, you are right, rather than `omitted`, I should use `invalid / valid`.
because those parameters are optional.

`fails when sortDirection parameter is invalid`
`succeeds when sortDirection valid parameters are provided`

what do you think about this?